aws/tailers/stack_event.go

```diff
@@ -9,22 +10,51 @@ import (
 	"text/tabwriter"
 
 	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/configservice"
 	"github.com/fatih/color"
 	"github.com/wallix/awless/aws/services"
 )
 
+const (
+	FilterStackEventLogicalID    = "id"
+	FilterStackEventTimestamp    = "ts"
+	FilterStackEventStatus       = "status"
+	FilterStackEventStatusReason = "reason"
+	FilterStackEventType         = "type"
+
+	// valid stack status codes
+	// http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html#w2ab2c15c15c17c11
+	cfStackEventCompleteSuffix   = "_COMPLETE"
+	cfStackEventFailedSuffix     = "_FAILED"
+	cfStackEventInProgressSuffix = "_IN_PROGRESS"
+)
```

Comment:
Could simply be:
```go
EventComplete   = "COMPLETE"
EventFailed  = "FAILED"
EventProgress = "IN_PROGRESS"
```
... clearer and they will still play the role of suffixes
```

Reply:
The same here.
Could be confusing which event is it if we add more tailors to the package.

Reply:
`StackEventComplete`